xtask/src/codegen/aya.rs

```diff
@@ -79,6 +79,7 @@ fn codegen_bindings(opts: &SysrootOptions, libbpf_dir: &Path) -> Result<(), anyh
         "bpf_lpm_trie_key",
         "bpf_cpumap_val",
         "bpf_devmap_val",
+        "bpf_stats_type",
```

Comment:
Please open a PR with just this change in it... We have a CI job that updates the generated code since we've had issues with user-submitted patches to generated code in the past.

Reply:
Updated in PR #988 🐱